src/TConsole.cpp

```diff
@@ -103,6 +118,9 @@ TConsole::TConsole(Host* pH, ConsoleType type, QWidget* parent)
             mMainFrameRightWidth = mpHost->mBorderRightWidth;
             mCommandBgColor = mpHost->mCommandBgColor;
             mCommandFgColor = mpHost->mCommandFgColor;
+
+            setAccessibleName(tr("Main Window"));
+            setAccessibleDescription(tr("Game messages are shown here"));
```

Comment:
Ah, I wasn’t aware that there could be sub-consoles. Is the following better?

`setAccessibleDescription(tr("Game content is shown here. It may contain subconsoles and a mapper window."));`

Reply:
Not sure that will be helpful to repeat this very often, because this text will be read very often. Let's wait until we have feedback that this is necessary before adding something potentially unnecessary.